en/micro-integrator/docs/administer-and-observe/working-with-monitoring-dashboard.md

```diff
@@ -1,67 +1,265 @@
 # Micro Integrator Dashboard
 
-The Micro Integrator dashboard provides a graphical view of the synapse artifacts that are deployed in a specified Micro Integrator server instance. This dashboard is an alternative to the [Micro Integrator CLI](../../administer-and-observe/using-the-command-line-interface), which allows you to monitor your deployments from the command line.
+The Micro Integrator dashboard provides a graphical view of the synapse artifacts that are deployed in a specified Micro Integrator server instance. The dashboard (as well as the [Micro Integrator CLI](../../administer-and-observe/using-the-command-line-interface)) communicates with the management API of WSO2 Micro Integrator to function.
```

Comment:
Currently we can do some user management and config management ( like changing log levels ) .. can we reflect that too somehow